layouts/partials/event_card.html

```diff
@@ -1,3 +1,29 @@
-<div class="event_card">
-  <h2>{{ .Title }}</h2>
+<link rel="stylesheet" href="../../assets/scss/components/_event_card.scss" />
+<link rel="preconnect" href="https://fonts.gstatic.com" />
+<link
+  href="https://fonts.googleapis.com/css2?family=Nunito:wght@600&display=swap"
+  rel="stylesheet"
+/>
+
+<div class="outer_card">
+  <div class="date_card">
+    <h3 style="text-align: center;">{{ .Params.date.Format "Jan" | upper }}</h3>
+    <h2 style="text-align: center;">{{ .Params.date.Format "2" }}</h2>
+    <h3 style="text-align: center;">{{ .Params.date.Format "2006" }}</h3>
```

Comment:
[Question]

What is the Jan 2 2006 date?

Reply:
According to the [Hugo dateFormat documentation](https://gohugo.io/functions/dateformat/) this syntax is how you're able to grab pieces of a date from a normal date string. Jan 2 2006 isn't any specific date and I'm not sure if the same functionality can be achieved using something other than this.